src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -0,0 +1,59 @@
+'''
+This module contains classes, methods & helpers that are used to get statistics
+(specifically number of files and total size of data present under the source
+and destination directory for the copy operation that is performed for snapshot
+cloning) and pass, print, log and convert them to human readable format
+conveniently.
+'''
+from typing import Optional
+
+from mgr_util import format_bytes, format_dimless
+
+
+def get_size_ratio_str(size1, size2, human=True):
+    if human:
+        size1, size2 = format_bytes(size1, 4), format_bytes(size2, 4)
+
+    size_string =  f'{size1}/{size2}'
+    size_string = size_string.replace(' ', '')
+    return size_string
+
+
+def get_num_ratio_str(num1, num2, human=True):
+    if human:
+        num1, num2 = format_dimless(num1, 4), format_dimless(num2, 4)
+
+    num_string = f'{num1}/{num2}'
+    num_string = num_string.replace(' ', '')
+    return num_string
+
+
+def get_amount_copied(src_path, dst_path, fsh, human=True):
+    rbytes = 'ceph.dir.rbytes'
+
+    size_t = int(fsh.getxattr(src_path, rbytes))
+    size_c = int(fsh.getxattr(dst_path, rbytes))
+
+    percent: Optional[float]
+    if size_t == 0 or size_c == 0:
+        percent = 0
+    else:
+        percent = ((size_c/size_t) * 100)
+        percent = round(percent, 3)
+
+    return size_t, size_c, percent
+
+
+def get_stats(src_path, dst_path, fsh, human=True):
```

Comment:
Preferred `fsh` over `fs_handle` it required lesser typing effort and more succinct (we use `fsp` in MDS code in same way), but I've replaced it.

Reply:
Sorry, I am having a hard time making you understand that smaller variable/parameter names are not succinct nor are `extra_long_paramter_names_that_look_like_this` useful.  The argument about MDS using `fsp` is probably because it's C++ and has a type associated with it, so you know what you are dealing with.